client/js/libs/handlebars/localedate.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
```

Comment:
You mean moving that helper into that file or using the localetime function to get the date? If the latter, that's going to be a bit more work because I'd need to localetime(date).split(",")[0]